src/media/css/app-list.styl

```diff
@@ -106,6 +103,31 @@ $desktop-tile-padding = 20px 50px;
     }
 }
 
+.filters-content {
```

Comment:
Can we put the app list filter-related styles into their own stylesheet? They can be their own component since they're separate from the actual app lists.